vertx-core/src/main/java/io/vertx/core/http/HttpClient.java

```diff
@@ -113,4 +118,12 @@ default Future<Void> close() {
    */
   Future<Void> shutdown(long timeout, TimeUnit unit);
 
+  /**
+   * Provides a way for users of the client to get access to the default redirect handler.
+   * This is useful for building redirect handlers that need to delegate to the default behavior.
+   */
+  @GenIgnore
+  default Function<HttpClientResponse, Future<RequestOptions>> defaultRedirectHandler() {
```

Comment:
rather an instance of on `HttpClientAgent` instead, in Vert.x 5, HttpClient is the API for interacting with an HTTP server whereas `HttpClientAgent` is the client.

Reply:
Good to know! PR updated